core/templates/dev/head/collection_player/CollectionPlayer.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for the learner's view of a collection.
+ *
+ * @author henning.benmax@gmail.com (Ben Henning)
+ */
+
+oppia.constant(
+  'COLLECTION_DATA_URL', '/collectionhandler/data/<collection_id>');
+
```

Comment:
Not immediately, but I noticed something strange when trying to debug this in the browser– apparently, the DOM has a bunch of invisible copies of the section titled 'Upcoming Explorations', and they seem to be doing some style changing when the animation is happening. Do you see this in your browser as well? Are these copies still there if the animation is removed?